werk24/models/property/stress_birefringence.py

```diff
@@ -30,7 +31,20 @@ class W24PropertyStressBirefringenceIso10110Grade(W24PropertyStressBirefringence
     grade: str = Field(examples=["SB20", "SB02"])
 
 
+class W24PropertyStressBirefringenceSchottGrade(W24PropertyStressBirefringence):
+    """Stress Birefringence Schott Grade"""
+    property_subtype: Literal["SCHOTT_GRADE"] = "SCHOTT_GRADE"
+    blurb: str = Field(examples=["Fine Annealed"])
+    grade: str = Field(examples=["Fine Annealed"])
+
```

Comment:
Make that a blurb. Its not an official schott grade. Its internal to them. If you start this here, youll end up implementing all customer-internal grades.

Reply:
updated